packages/flutter_tools/lib/src/application_package.dart

```diff
@@ -28,6 +28,8 @@ abstract class ApplicationPackage {
 
   String get displayName => name;
 
+  String get packagePath => null;
```

Comment:
Ah, only overridden for APKs. Perhaps make this method abstract and have all clients implement (even with `=> null` implementations)?